meta-cli/src/cli/codegen.rs

```diff
@@ -48,29 +44,29 @@ pub struct Deno {
 
 #[async_trait]
 impl Action for Deno {
-    async fn run(&self, args: GenArgs) -> Result<()> {
-        let dir = args.dir()?;
-        // try to find config file, else use default config as the options
-        // used for code generation have default values.
-        let config = Arc::new(
-            Config::load_or_find(args.config, &dir).unwrap_or_else(|_| Config::default_in(&dir)),
-        );
+    async fn run(&self, _args: GenArgs) -> Result<()> {
+        // TODO:
+        // how does this fit with the current impl?
```

Comment:
Let's move this under the `meta gen` command? It'd be another module for `metagen`.